app/locales/en/translations.ts

```diff
@@ -547,11 +551,14 @@ export default {
                 consent: 'I have read these terms. I understand that articles must be published by December 31, 2018, in order to be eligible for a prize.',
                 continue: 'Continue',
             },
-            page_title: '{{nodeTitle}} Registrations',
+            page_title: '{{{nodeTitle}}} Registrations',
             no_registrations: 'There have been no completed registrations of this project.',
+            only_admins_can_initiate: 'Only project administrators can initiate registrations.',
             no_drafts: 'There are no draft registrations of this project.',
             start_new: 'Start a new registration by clicking the “New registration” button. Once created, registrations cannot be edited or deleted.',
             learn_more: 'Learn more about registrations <a href="{{learnMoreLink}}">here</a>.',
+            register_entire_project: 'To register the entire project "{{{rootNodeTitle}}}" instead, click',
```

Comment:
I think you're right the page title helper does its own sanitizing, but this one puts the desanitized title straight in the dom.

I named a project `&lt;img src="x" onerror="alert('yay')"&gt;` (alas, the browser was smart enough not to run my `&lt;script&gt;` tag), then went to the registrations tab of a child node:
<img width="470" alt="screen shot 2018-10-04 at 17 22 19" src="https://user-images.githubusercontent.com/6776190/46503986-8d3aea00-c7fa-11e8-8824-47aed591f644.png">

We need to be really careful with `{{{ }}}`, and I might actually consider it a bug that the title helper requires it to work as expected.

Reply:
I solved this a safer (IMHO) way: https://github.com/CenterForOpenScience/ember-osf-web/pull/417/commits/4ec054d9ded4b060c06af06fc4088fb4ae131ce6